### PR TITLE
Maid 1344 low balance error

### PR DESCRIPTION
**Pull request by benjaminbollen**
[Review on Reviewable](https://reviewable.io/reviews/maidsafe/routing/671)

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
r? @brian-js

(maidsafe_highfive has picked a reviewer for you, use r? to override)

**Comment by benjaminbollen** [comment]
@maqi this affects your API as well in routing 0.3.9

**Comment by ustulation** [comment]
seems fine @benjaminbollen

File: src/error.rs [hunk 1]
```diff
@@ -32,6 +32,8 @@ pub enum ResponseError {
     /// Abort is for user to indicate that the state can be dropped;
     /// if received by routing, it will drop the state.
     Abort,
+    /// On low balance or no account registered
+    LowBalance(Data, u32),
     /// invalid request
```
Comment (maqi) [inline src/error.rs]: I though NoAccount was agreed to be put in , though it won't be hit during the test?

**Pull request merged**